watch/plan.go

```diff
@@ -20,10 +20,12 @@ const (
 )
 
 // Run is used to run a watch plan
-func (p *Plan) Run(address string) error {
+func (p *Plan) Run(address string, conf *consulapi.Config) error {
```

Comment:
It might be worth keeping this signature the same and moving to a RunWithConfig method so we don't break tools that import this package. It is pretty small, though - what do you think?

Reply:
Thats probably best. Nothing in our code does but I guess the api can be imported by other tools.